CHANGELOG.md

```
@@ -1,528 +1,266 @@
-* remove triggering the dev env setup on every pull request and pushes ([c85375d](https://github.com/raghavyuva/nixopus/commit/c85375da27771d9dc8a6294ef5c1896ba9906ccb))
-* remove version comparision check ([240716c](https://github.com/raghavyuva/nixopus/commit/240716c4e7dbb95315506c5b1f3b02dbaba97a3d))
-* removed docker compose dependency ([d680381](https://github.com/raghavyuva/nixopus/commit/d68038149f73656f4be5fa4a9034e374518c0994))
-* removed go installation and auto installation of docker git etc deps as it may cause errors and conflicts ([ed9207f](https://github.com/raghavyuva/nixopus/commit/ed9207fc64088d530cbe756ba7adab7a3180b76b))
-* seperate jobs for domain based installation and ip based installation ([b0736ad](https://github.com/raghavyuva/nixopus/commit/b0736ad5fd31c65aeb87d5157a19e282fdcaaeb9))
-* unsupported architecture golang install ([e96271c](https://github.com/raghavyuva/nixopus/commit/e96271cfa7bbfd568f508cbae9e75f84493a59e9))
-* update complete.yml to use the split imports for different fixtures, and add custom support for importing the fixtures using gopkg/yaml ([01e587f](https://github.com/raghavyuva/nixopus/commit/01e587f8aab667427d298ddec00163f4f455ccd0))
-* uses logging module instead of print_debug function for extendability for future changes and to keep consistency ([f709e45](https://github.com/raghavyuva/nixopus/commit/f709e4509befbd71e119a6efd37f339dd927df66))
+# [0.1.0-alpha.32](https://github.com/pankajydv07/nixopus/compare/v0.1.0-alpha.31...v0.1.0-alpha.32) (2025-09-27)
 
 
 ### Features
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)